pkg/cfn/builder/nodegroup.go

```diff
@@ -122,7 +124,10 @@ func (n *NodeGroupResourceSet) newResource(name string, resource gfn.Resource) *
 
 func (n *NodeGroupResourceSet) addResourcesForNodeGroup() error {
 	launchTemplateName := gfnt.MakeFnSubString(fmt.Sprintf("${%s}", gfnt.StackName))
-	launchTemplateData := newLaunchTemplateData(n)
+	launchTemplateData, err := newLaunchTemplateData(n)
+	if err != nil {
+		return errors.Wrap(err, "couldn't add resources for nodegroup")
```

Comment:
nit: i don't think we use contraction in error messages (or at least we shouldn't)

```suggestion
		return errors.Wrap(err, "could not add resources for nodegroup")
```